containers/go/README.md

```diff
@@ -20,6 +23,34 @@ While the definition itself works unmodified, you can select the version of Go t
 "args": { "VARIANT": "1.14" }
 ```
 
+You can also directly reference pre-built versions of `.devcontainer/base.Dockerfile` by using the `image` property in `.devcontainer/devcontainer.json` or updating the `FROM` statement in your own  `Dockerfile` to one of the following. An example `Dockerfile` is included in this repository.
+
+- `mcr.microsoft.com/vscode/devcontainers/go` (latest)
+- `mcr.microsoft.com/vscode/devcontainers/go:1`
+- `mcr.microsoft.com/vscode/devcontainers/go:1.15`
+- `mcr.microsoft.com/vscode/devcontainers/ruby:1.14`
```

Comment:
ruby? is it just a coincidence or a real error?

Reply:
That's a typo.  Good catch!  Copy-paste from the ruby definition and I missed one. :smile:

I'll remove `1.13` then.

Going forward, we'll just need to update the "variants" property in definition-manifest.json and the README as new versions come out and the previous one is deprecated. The rest should stay the same.